### DISCUSSION TITLE
SkyNode optimization problem

### DISCUSSION THREAD

**Discussion by XiaobaiSW** [pelicanmapping/osgearth#2322] [Support Q&A] [answered]
In osgEarth version 2.7, when loading a skybox, I've noticed a white border appearing around the Earth due to the lighting. The border seems a bit wider than desired. Is there a way to optimize (narrow) this border or remove it? I've observed that Google Earth doesn't seem to have this issue, and I'd like to achieve a similar effect.

I appreciate any guidance or suggestions you can provide to address this concern. Thank you!
![image](https://github.com/gwaldron/osgearth/assets/47914115/030194de-89ab-4a70-b28c-d1dd0d84be37)
![image](https://github.com/gwaldron/osgearth/assets/47914115/c1a621bf-4fc7-4b38-8f23-a3d7fc0cd77b)

**Comment by gwaldron** [comment, marked as answer]
There is a function `SkyNode::setAtmosphereVisible` to toggle the atmosphere, but it did not exist in version 2.7.

**Comment by XiaobaiSW** [comment]
Thanks!
